spec/example_app/app/dashboards/order_dashboard.rb

```diff
@@ -14,6 +14,8 @@ class OrderDashboard < Administrate::BaseDashboard
     line_items: Field::HasMany,
     total_price: Field::Number.with_options(prefix: "$", decimals: 2),
     shipped_at: Field::DateTime,
+    status: Field::Enum.with_options(
+      enum: { "pending" => 0, "confirmed" => 1 }),
```

Comment:
I'm not thrilled with needing to pass in the enum values to the field, because it's duplicating the logic between the models and the dashboard. I'm not sure I know of a better way, though.